Plan/api/src/main/java/com/djrapitops/plan/extension/builder/ValueBuilder.java

```diff
@@ -162,6 +160,14 @@ default ValueBuilder formatAsTimeAmount() {
      */
     DataValue<String> buildString(String value);
 
+    /**
+     * Build a {@link Component}.
+     *
+     * @param value a {@link Component} made by {@link com.djrapitops.plan.component.ComponentService}
+     * @return a data value to give to {@link ExtensionDataBuilder}.
+     */
+    DataValue<String> buildComponent(String value);
```

Comment:
These probably could use Component instead to be clear for api users :)

Reply:
Just the parameter or the DataValue as well?

Reply:
I think both makes sense since that would match how the other ones work